src/server.rs

```diff
@@ -167,7 +167,7 @@ impl Default for Server<()> {
 }
 
 impl<State: Send + Sync + 'static> Server<State> {
-    /// Create a new Tide server with shared application scoped state.
+    /// Create a new Tide server with application scoped state.
```

Comment:
Application state is and always was shared within the application. Now that `State: Clone` it's true both locally to applications, and between applications.

```suggestion
    /// Create a new Tide server shared with application scoped state.
```